common/k8stest/util_rdma.go

```diff
@@ -317,3 +317,41 @@ func GetDevLinkName(node, iface string) (string, error) {
 	}
 	return "", nil
 }
+
+func DisableRdmaDeicesOnAllMayastorNodes() error {
+	workerNodes, err := ListIOEngineNodes()
+	if err != nil {
+		return err
+	}
+	for _, node := range workerNodes.Items {
+		logf.Log.Info("Disable rdma device", "Node", node.Name)
+		ifaceList := e2e_config.GetConfig().DisabledRdmaDevices
+		for _, iface := range ifaceList {
+			err := DisableRdmaOnNode(node.Name, iface)
+			if err != nil {
+				return err
+			}
+		}
+
+	}
+	return nil
+}
+
+func EnableRdmaDeicesOnAllMayastorNodes() error {
+	workerNodes, err := ListIOEngineNodes()
```

Comment:
IMO should return here immediately if e2e_config.GetConfig().DisabledRdmaDevices is empty